annotations.md

```diff
@@ -40,6 +40,14 @@ This specification defines the following annotation keys, intended for but not l
     ```
 * **org.opencontainers.image.title** Human-readable title of the image (string)
 * **org.opencontainers.image.description** Human-readable description of the software packaged in the image (string)
+* **org.opencontainers.image.base.digest** [Digest](descriptor.md#digests) of the image this image is based on (string)
+  * This SHOULD be the immediate image sharing zero-indexed layers with the image, such as from a Dockerfile `FROM` statement.
```

Comment:
I think I'd be happier with this line being `MAY`.

Reply:
To your point in [the meeting](https://www.youtube.com/watch?v=gFZQplmiSDc), if it's MAY, we should also recommend that clients consuming it SHOULD fetch that image by digest to try to understand what kind of relationship it has to this image. If it shares zero-indexed layers, that means something in particular. If not, it means, uh... 🤷

Reply:
Ah, I should have read this before writing [my response](https://github.com/opencontainers/image-spec/pull/822#issuecomment-803098986). Yes, I agree.

Reply:
When you say, fetch by digest, the question is whether this value will have just the digest, or the registry/repo to find the digest.
What I thought this meant was:
- `org.opencontainers.image.base.ref.name=registry.wabbit-networks.io/net-monitor:v1`
- `org.opencontainers.image.base.digest=sha256:e752324f6804d5d0b2c098f84507d095a8fd0031cf06cdb3c7ad1625dcd1b399`

What the above could mean is:
I can monitor the base image for changes. If the digest of the ref.name doesn't equal the digest, I know it changed and I may want to rebuild.

But, I'm not reading that in this description, so perhaps your thinking these are used for another reason.

The general framing I have around annotations is:
- is the value intended for humans to read/parse and possibly do something
- is the value intended for computers to parse, which should be very specific. We shouldn't need an AI/ML process to iterate all the possible permutations to understand the intent.

If the value is for humans, it's less important, because humans are very good about inferring meaning, and there's no automation. If it's for automation, we should be very specific and actionable for public software and any user to automate around. 

If it's for specific companies, than Acme Rockets can create their own annotations and use them any way they'd like. 
Putting the grammar info in the ref-name seems less important than an actual example, as the example implies whether this is intended to be a fully qualified registry name (docker.io/library/ubuntu), a default docker hub reference (ubuntu - please don't as we need to kill the default docker hub concept), or a human value that says `ubuntu` but is only interesting for the reader to think, oh, that's the ubuntu image, but I don't know where exactly they got it from, or where I monitor if it's still this digest.